package.json

```diff
@@ -1,87 +1,93 @@
 {
   "name": "react-oui-icons",
-  "version": "1.0.1",
-  "main": "lib/react-oui-icons.js",
+  "version": "2.0.0",
+  "main": "index.js",
```

Comment:
Shouldnt this be the babel built file? `/build/index.js`

Reply:
yeah, we do want to reference the babel built file, but once we copy this package.json into the build folder they will be in the same folder